routes/seller.js

```diff
@@ -0,0 +1,182 @@
+var express = require('express');
+var router = express.Router();
+const { check, validationResult } = require('express-validator');
+const uuidv4 = require('uuid/v4');
+var body = require('express-validator'); //validation
+var moment = require('moment'); //datetime
+const multer = require('multer'); //middleware for handling multipart/form-data, which is primarily used for uploading files
+const upload = multer({ dest: './static/images/produce_images/' }); //path.join(__dirname, 'static/images/produce_images/)
+var ROLES = require('../utils/roles');
+var fs = require('fs');
+const axios = require('axios');
+const crypto = require('crypto');
+const hash = crypto.createHash('sha256');
+
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args));
+
+var initModels = require('../models/init-models');
+var sequelise = require('../config/db/db_sequelise');
+
+var models = initModels(sequelise);
+
+//GET seller dashboards
+router.get(
+  '/',
+  require('connect-ensure-login').ensureLoggedIn({ redirectTo: '/app/auth/login' }),
+  function (req, res, next) {
+    if (req.user.role === ROLES.Seller) {
```

Comment:
I will merge this shortly but let's add an admins and superusers here i.e. in addition to sellers